afdko/Tools/Programs/spot/source/opt.c

```diff
@@ -285,7 +285,7 @@ int opt_##n(int argc, char *argv[], int argi, opt_Option *opt) \
 	else \
 		{ \
 		t value; \
-		char s[64]; \
+		char s[128]; \
```

Comment:
Should `NAME_LEN` or `MAX_NAME_LEN` be used here?

Reply:
Didn't mean to change this. I will revert it. There is no dependency on  glyph name length here.  This code is for parsing the  arguments. 's' holds values for arguments, like "GPOS=6". There is only one potentially long string value. This is the list of glyph ID's used with the  command "spot -t CFF=6 -g <list of glyph ID's>. This  cannot take glyph names, or I would have had to allocate memory dynamically. The logic keeps the buffer from being overrun by truncating the input string, if it is too long.